add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $title = $_POST['title'];
+    $author = $_POST['author'];
+    $year = $_POST['year'];
+
+    $query = "INSERT INTO books (title, author, year) VALUES ('$title', '$author', $year)";
+    if (mysqli_query($connection, $query)) {
+        echo "Book added successfully!";
+    } else {
+        echo "Error: " . mysqli_error($connection);
+    }
+}
+?>
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Add Book</title>
+</head>
```

Comment:
Validate user inputs to ensure they are not empty and are of the correct type (e.g., year should be a number).